src/index.html

```diff
@@ -17,11 +20,436 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+
+      <nav class="header__nav">
```

Comment:
Nav can be a separate block, even now it looks not like element of header, so please split it in naming convension